### ISSUE TITLE
Les données d'actions ne sont pas toujours complétées

### ISSUE THREAD

**Issue by VincentAntoine** [MTES-MCT/monitorenv#1629]

**Comment by VincentAntoine** [comment]
@AdelineCelier @claire2212 @maximeperraultdev

**Comment by maximeperraultdev** [comment]
@VincentAntoine Hello, on a remarqué que les id des infractions et des actions lors de la duplication étaient identiques. La PR qui devrait corrigé cela est [mergée](https://github.com/MTES-MCT/monitorenv/pull/1624) et sera livrée avec la version v1.40.0 livré prochainement. Je vais également faire un ticket pour migrer les anciennes données afin de leur donner un identifiant différent pour chaque.

**Comment by VincentAntoine** [comment]
@maximeperraultdev Ce n'est pas le même sujet je crois :upside_down_face: Il s'agit ici statut `completion`, je ne pense que ce soit lié au sujet des ids dupliqués ?

**Comment by VincentAntoine** [comment]
Je viens de voir que ce sont très majoritairement les `NOTE` qui sont en statut `completion=TO_COMPLETE`.

![image](https://github.com/user-attachments/assets/d4fd1dc9-6b46-4d5c-acaf-345b6ff5dcef)

Les `NOTE` sont de toutes façons exclues de la vue `analytics_actions` et il n'y a pas de stats dessus donc çan'a pas d'impact en réalité, Mais ça peut être source de confusion. A l'occasion ça peut être cool de cleaner. Pour les NOTE il semblerait plus logique que le champ completion soit systématiquement `COMPLETED` ou `NULL` plutôt que `TO_COMPLETE`.

**Comment by AdelineCelier** [comment]
Hello, @VincentAntoine est-ce que si l'on exclut le souci lié aux notes (qui en effet ne devraient pas être comptabilisées dans les actions à compléter), il reste beaucoup d'actions non complétées ? Y en a-t-il suffisamment pour justifier d'ajouter des rappels supplémentaires dans l'app ?

**Comment by maximeperraultdev** [comment]
@AdelineCelier pour moi ça ressemble à un bug. J'ai créé un ticket https://github.com/MTES-MCT/monitorenv/issues/1636

**Comment by AdelineCelier** [comment]
Yes, le fait que les notes soient en "to complete" est effectivement un bug. 
Mais je demandais à Vincent si en-dehors de ces notes il y avait d'autres actions en "to complete" et combien, et si ça vaut le coup de mettre qque chose en place pour ces autres actions.

**Comment by VincentAntoine** [comment]
OK en inté

**Comment by VincentAntoine** [comment]
@AdelineCelier En prod, pour ce qui est des contrôles et surveillances TO_COMPLETE, voici ce qu'il y a :

![image](https://github.com/user-attachments/assets/f73c735e-8ab4-4066-87d1-897add70236e)

Donc pas beaucoup au regard du total, une dizaine d'actions restent en TO_COMPLETE sur ~800 actions par mois, mais peut-être un process de clean mensuel à mettre en place ?

**Comment by AdelineCelier** [comment]
@VincentAntoine pardon pour le temps de réponse, le ticket était passé à la trappe ! 
Oui je pense qu'un process de clean par le CACEM peut suffire, sachant que c'est assez facile dans l'app de filtrer par "données à compléter".
